SU-SWS/stanford_profile
# D8CORE-642: Stanford Card Link display option.

# READY FOR REVIEW

# Summary
- Adds a drop-down to allow the user to choose a display option for the link.

# Review By (Date)
- October 29

# Urgency
- Medium. Nice to have for the Pilot.

# Steps to Test

1. Check out this branch.
2. Do a full install `blt drupal:install`
3. Log in and edit the home page node
4. Edit the Card paragraph and validate that the link display drop-down is available
5. Choose the `Action Link` option and save the node
6. Validate that the card displays a link
7. Edit the card again and choose `Button`
8. Save the node again and validate that it is a button.
9. Run tests

# Affected Projects or Products
- Stanford Page
- Stanford Card Paragraph
- D8CORE
- Jackalope

# Associated Issues and/or People
- D8CORE-642

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
